app/javascript/packs/flagUserModal.js

```diff
@@ -0,0 +1,146 @@
+const modalSnackbarHTML = `
+<div class="crayons-snackbar">
+  <div class="crayons-snackbar__item" id="vomit-all-snackbar">
+    <div class="crayons-snackbar__body">
+      <p>All posts by this author will be less visible.</p>
+    </div>
+  </div>
+  <div class="crayons-snackbar__item" id="abuse-report-snackbar">
+    <div class="crayons-snackbar__body">
+      <p>Thanks for the report. Our team will review this.</p>
+    </div>
+  </div>
+</div>
+`;
+
+const toggleFlagUserModal = () => {
+  document
+    .querySelector('.flag-user-modal-container')
+    .classList.toggle('hidden');
+};
+
+const flashSnackbar = (snackbar) => {
```

Comment:
I thought @nickytonline had said that the snackbar has disappear functionality, but my snackbar was not disappearing, so I added that behavior.